.gitignore

```diff
@@ -1,4 +1,3 @@
 .qmake.stash
 Makefile
-webui/*
```

Comment:
As a know about the RSwebui, the WebUI folder is the bundling of all the files of the `webui-src/app` which creates every time when we build. So, Do your changes in `webui-src /app`  folder. No need to push the `webui` folder. so please add `webui/*` in `.gitignore`.

Reply:
There are changes in app.js that were not being pushed because app.js was in git ignore and without app.js, classes are not getting applied, so is there any other alternative?

Reply:
Hey @gunjan. Once look at the app.js codebase, it is exactly the same code of  `WebUI-src/app` files. So, Do your changes to that files. As I already mentioned `webui` folder is bundling of the `webui-src/app`. You can also check it in `webui-src/make-src/build.*`

Reply:
Ok, got it ! Thankyou